include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,553 @@
+    //VkBool32           transformFeedback;
+    //VkBool32           geometryStreams;
+
+    /* VertexAttributeDivisorFeaturesEXT *//* VK_EXT_vertex_attribute_divisor */
+    //VkBool32           vertexAttributeInstanceRateDivisor;
+    //VkBool32           vertexAttributeInstanceRateZeroDivisor;
+
+    /* VertexInputDynamicStateFeaturesEXT *//* VK_EXT_vertex_input_dynamic_state */
+    //VkBool32           vertexInputDynamicState;
+
+    /* Ycbcr2Plane444FormatsFeaturesEXT *//* VK_EXT_ycbcr_2plane_444_formats */
+    //VkBool32           ycbcr2plane444Formats;
+
+    /* YcbcrImageArraysFeaturesEXT *//* VK_EXT_ycbcr_image_arrays */
+    //VkBool32           ycbcrImageArrays;
+
+    /* ShaderIntegerFunctions2FeaturesINTEL *//* VK_INTEL_shader_integer_functions2 */
+    //VkBool32           shaderIntegerFunctions2;
+
+    /* 16BitStorageFeaturesKHR *//* VK_KHR_16bit_storage *//* MOVED TO Vulkan 1.1 Core */
+    /* 8BitStorageFeaturesKHR *//* VK_KHR_8bit_storage *//* MOVED TO Vulkan 1.2 Core */
+
+    /* AccelerationStructureFeaturesKHR *//* VK_KHR_acceleration_structure */
+    bool accelerationStructure = false;
+    bool accelerationStructureCaptureReplay = false;
+    bool accelerationStructureIndirectBuild = false;
+    bool accelerationStructureHostCommands = false;
+    bool descriptorBindingAccelerationStructureUpdateAfterBind = false;
+            
+    /* BufferDeviceAddressFeaturesKHR *//* VK_KHR_buffer_device_address *//* MOVED TO Vulkan 1.2 Core */
+
+    /* DynamicRenderingFeaturesKHR *//* VK_KHR_dynamic_rendering *//* MOVED TO Vulkan 1.3 Core */
+
+    /* [!!NV Version below, struct doesn't exist in vk headers] VK_KHR_fragment_shader_barycentric */
+
+    /* FragmentShadingRateFeaturesKHR *//* VK_KHR_fragment_shading_rate */
+    //VkBool32           pipelineFragmentShadingRate;
+    //VkBool32           primitiveFragmentShadingRate;
+    //VkBool32           attachmentFragmentShadingRate;
+    
+    /* GlobalPriorityQueryFeaturesKHR *//* VK_KHR_global_priority */
+    //VkBool32           globalPriorityQuery;
+
+    /* ImagelessFramebufferFeaturesKHR *//* VK_KHR_imageless_framebuffer *//* MOVED TO Vulkan 1.2 Core */
+    /* Maintenance4FeaturesKHR *//* VK_KHR_maintenance4 *//* MOVED TO Vulkan 1.3 Core */
+    /* MultiviewFeaturesKHR *//* VK_KHR_multiview *//* MOVED TO Vulkan 1.1 Core */
+
+    /* PerformanceQueryFeaturesKHR *//* VK_KHR_performance_query */
+    //VkBool32           performanceCounterQueryPools;
+    //VkBool32           performanceCounterMultipleQueryPools;
+
+    /* PipelineExecutablePropertiesFeaturesKHR *//* VK_KHR_pipeline_executable_properties */
+    //VkBool32           pipelineExecutableInfo;
+    
+    /* VK_KHR_portability_subset - PROVISINAL/NOTAVAILABLEANYMORE */
+
+    /* PresentIdFeaturesKHR *//* VK_KHR_present_id */
+    //VkBool32           presentId;
+
+    /* PresentWaitFeaturesKHR *//* VK_KHR_present_wait */
+    //VkBool32           presentWait;
```

Comment:
won't expose, this extension is shit, I should have a Fence-andQuery-like object to query the presentation timestamp, not a blocking call that may unblock after an arbitrary delay from the present